libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.repository.ts

```diff
@@ -1,58 +1,64 @@
 import { Injectable } from '@nestjs/common'
 import { logger } from '@island.is/logging'
+import {
+  UnionApi,
+  Union,
+  PensionApi,
+  PensionFund,
+} from '@island.is/vmst-client'
 import { ParentalLeavePeriod } from './parentalLeavePeriod.model'
-import { Union } from './union.model'
-import { PensionFund } from './pensionFund.model'
 import { ParentalLeaveEntitlement } from './parentalLeaveEntitlement.model'
 import { ParentalLeavePaymentPlan } from './parentalLeavePaymentPlan.model'
 
-// const accessToken = process.env.DIRECTORATE_OF_LABOUR_ACCESS_TOKEN
-
-// TODO implement this class when the endpoints are ready, this is just mocks atm
+const isRunningInDevelopment = process.env.NODE_ENV === 'development'
 
 @Injectable()
 export class DirectorateOfLabourRepository {
-  //   private client: any
-
-  constructor() {
+  constructor(private unionApi: UnionApi, private pensionApi: PensionApi) {
     logger.debug('Created Directorate of labour repository')
   }
 
-  //   getClient(): any {
-  //     if (!accessToken) {
-  //       throw new Error(
-  //         'Missing environment variables: DIRECTORATE_OF_LABOUR_ACCESS_TOKEN',
-  //       )
-  //     }
+  async getUnions(): Promise<Union[]> {
+    if (isRunningInDevelopment) {
+      return [
+        {
+          id: 'id',
+          name: 'VR',
+        },
+      ]
+    }
```

Comment:
Not sure if this belongs here. Should either have a "simple guide" to connect to dev xroad, or use api/mocks and put this there.

Reply:
Will refactor in another PR.